my_game.py

```diff
@@ -28,10 +28,9 @@
 PLAYER_LIVES = 3
 PLAYER_SPEED = 3
 PLAYER_SHOT_SPEED = 4
-PLAYER_THRUST = 0.5
```

Comment:
We need to keep that. That's how much speed is added to the player when thrust/rocket/engine is enabled.

Reply:
True, but the player cannot gain speed in other ways - so we remove the other player speed?

Reply:
self.speed is the current speed of the player. PLAYER_THRUST is the power of the rocket.